site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -133,33 +152,27 @@ gcloud compute firewall-rules create game-server-firewall \
 
 Create a GKE cluster in which you'll install Agones. You can use
 [GKE Standard mode](#create-a-standard-mode-cluster-for-agones)
-or [GKE Autopilot mode](#create-an-autopilot-mode-cluster-for-agones)
-(Alpha).
+or [GKE Autopilot mode](#create-an-autopilot-mode-cluster-for-agones).
 
 ### Create a Standard mode cluster for Agones
 
+Create the cluster:
 ```bash
-gcloud container clusters create [CLUSTER_NAME] --cluster-version={{% gke-example-cluster-version %}} \
+gcloud container clusters create [CLUSTER_NAME] \
+  --region=[COMPUTE_REGION]
```

Comment:
You need to add a `\`at the end of these two new lines so that the command can be copy/pasted.